modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/uuid/ItUuidDmlTest.java

```diff
@@ -17,71 +17,26 @@
 
 package org.apache.ignite.internal.sql.engine.datatypes.uuid;
 
-import static org.apache.ignite.internal.lang.IgniteStringFormatter.format;
-import static org.apache.ignite.internal.sql.engine.util.SqlTestUtils.assertThrowsSqlException;
-
 import java.util.UUID;
 import org.apache.ignite.internal.sql.engine.datatypes.DataTypeTestSpecs;
 import org.apache.ignite.internal.sql.engine.datatypes.tests.BaseDmlDataTypeTest;
 import org.apache.ignite.internal.sql.engine.datatypes.tests.DataTypeTestSpec;
 import org.apache.ignite.internal.sql.engine.datatypes.tests.TestTypeArguments;
 import org.apache.ignite.internal.sql.engine.type.UuidType;
-import org.apache.ignite.lang.ErrorGroups.Sql;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.MethodSource;
 
 /**
  * Tests for DML statements for {@link UuidType UUID data type}.
  */
 public class ItUuidDmlTest extends BaseDmlDataTypeTest<UUID> {
-    /** {@code INSERT} with dynamic parameters is not allowed for compatible types. */
-    @ParameterizedTest
-    @MethodSource("convertedFrom")
```

Comment:
Do we have another similar test for UUID?

Reply:
it used to check that implicit cast from string to uuid was not allowed for insert for some reason. Now it's not allowed in general, it's covered by `DifferentFamiliesInsertSourcesCoercionTest`